build/kube_setup.sh

```diff
@@ -1,8 +1,10 @@
 #!/bin/bash
+# Configures a local minikube cluster for testing.
 set -e
 
-# Script you can use to set up a local minikube cluster for testing
-# It is assumed you have already run 'minikube start' and your kubectl context is using the minikube cluster
+# Set to `kubectl` if kubectl is installed system-wide, its context is set to your minikube cluster,
+# and you wish to use it instead of the kubectl bundled with minikube.
+KUBECTL_CMD='minikube kubectl --'
```

Comment:
Doesn't this need to be something like `?=` to set it as a default but allow override if the env var is set? Plus, @bsquizz mentioned doing a check by default to see if kubectl is installed and available. I think the default should be, try to use `kubectl` from path if available, then fall back to `minikube`, and then finally if the user sets the env var, we should just override it with that.

Reply:
I think the lookup order would be:

1. Use $`KUBECTL_CMD`, and if it's unset...
2. Use `kubectl`, and if it's unavailable...
3. Use `minikube`.

Right?

Reply:
Yeah IMO that's the best path for the logic